### DISCUSSION TITLE
Will my private repo config override a package in PyPi with same name?

### DISCUSSION THREAD

**Discussion by wally-lima** [python-poetry/poetry#8863] [Q&A]
I have a private package in Azure DevOps artifacts with the same name as a public package in Pypi. My project installs packages from PyPi and this private repo. Will the setting for a private source override the package with the same name in PyPi? If it doesn't automatically override, how can I assure it will?

**Comment by radoering** [comment]
By setting the priority of your private repo (and maybe PyPI): https://python-poetry.org/docs/repositories/#project-configuration
